LICENSE

```diff
@@ -0,0 +1,21 @@
+MIT License
```

Comment:
I don't think you need this file since it's in the source distribution already.

Reply:
Done